app/models/approval_request.rb

```diff
@@ -4,4 +4,10 @@ class ApprovalRequest < ApplicationRecord
   enum :state => [:undecided, :approved, :denied]
 
   belongs_to :order_item
+
+  scope :by_owner, lambda {
```

Comment:
@lindgrenj6 if we are going to remove owner from order_item would we have to redo these scopes to get the owner from order?

Reply:
That's correct. we'll have to do a double join to get there. Off the top of my head the sql will look like this: 
```sql
SELECT approval_requests.*,order_items.order_id,orders.owner
FROM approval_requests
INNER JOIN order_items ON order_items.id = approval_requests.order_item_id
INNER JOIN orders ON orders.id = order_items.order_id;
```

From there the owner will be on the query just like it is here.

Reply:
I'll update these to look to the order now so they won't have to be updated then, rather than waiting.